packages/engine/errors/1109.md

```diff
@@ -0,0 +1,37 @@
+---
+original: "Expression expected."
+excerpt: "I was expecting your code to resolve to a value."
+---
+
+Your code is likely to be missing an expression which is preventing it from producing a value.
+
+### Examples
+
+
+`import {} from` 
```

Comment:
Could you convert the examples to a code block?
Inline code blocks are usually fine if they're on the same line:
````suggestion
```ts
import {} from
```
````